pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -415,13 +330,14 @@ func (pl *InterPodAffinity) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, state
 	return true, nil
 }
 
-//  nodeMatchesAllTopologyTerms checks whether "nodeInfo" matches topology of all the "terms" for the given "pod".
-func nodeMatchesAllTopologyTerms(pod *v1.Pod, topologyPairs topologyToMatchedTermCount, nodeInfo *framework.NodeInfo, terms []v1.PodAffinityTerm) bool {
+//  nodeMatchesAllAffinityTopologies checks whether "nodeInfo" matches the topologies of
+// all affinity terms of the incoming pod.
+func nodeMatchesAllAffinityTopologies(nodeInfo *framework.NodeInfo, state *preFilterState) bool {
```

Comment:
it checks both topologies' presence and terms' match, so prefer:

```suggestion
func nodeMatchesAllAffinityTerms(nodeInfo *framework.NodeInfo, state *preFilterState) bool {
```

Reply:
done